modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/info/item/provider/CPDefinitionInfoItemObjectProvider.java

```diff
@@ -51,16 +53,16 @@ public CPDefinition getInfoItem(InfoItemIdentifier infoItemIdentifier)
 		ClassPKInfoItemIdentifier classPKInfoItemIdentifier =
 			(ClassPKInfoItemIdentifier)infoItemIdentifier;
 
-		CPDefinition cpDefinition = _cpDefinitionLocalService.fetchCPDefinition(
-			classPKInfoItemIdentifier.getClassPK());
-
-		if (cpDefinition == null) {
+		try {
```

Comment:
why this was made that way? your version looks cleaner 👍

Reply:
I don't know why but without the throwable the exception was not handled correctly by the framework